examples/toy-model-stochastic-degradation.ipynb

```diff
@@ -0,0 +1,201 @@
+{
```

Comment:
Notebook looks lovely, but some minor cleaning up to be done (2 empty cells and one equation that isn't rendering)